src/lib/containers/UserSearchBox.tsx

```diff
@@ -3,48 +3,39 @@ import Downshift from 'downshift'
 import { useState } from 'react'
 import { getUserGroupHeaders } from '../utils/SynapseClient'
 import { UserGroupHeader, UserGroupHeaderResponsePage } from '../utils/synapseTypes'
-import { UserCardSmall } from './UserCardSmall'
+import UserCard from './UserCard'
+import { SMALL_USER_CARD } from '../utils/SynapseConstants'
+import IconSvg from './IconSvg'
+import {
+  TYPE_FILTER
+} from '../utils/synapseTypes/UserGroupHeader'
+
 
 export type UserSearchBoxProps = {
   id?: string  // id for the input tag
-  onSelectCallback?: (selected:FormattedUserHeader) => void,
+  onSelectCallback?: (selected:UserGroupHeader) => void,
+  typeFilter?: TYPE_FILTER,
```

Comment:
You can now configure the search box with a type filter (ALL, USERS_ONLY, or TEAMS_ONLY)